README.md

```diff
@@ -1,3 +1,17 @@
+# Actions - GitHub Actions for Azure
+
+This repository provides a framework, guidleines and processes to author new and contribute to existing GitHub Actions deploying to Azure.
+
+## Table of contents
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- Contributing to Azure Actions
+  - [Create a new GitHub Action for Azure or Microsoft](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#creating-a-new-github-action-for-azure-or-microsoft)
+  - [Guidelines for repository permissions](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#guidelines-for-setting-permissions-on-the-repo)
+  - [Action Versioning](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#action-versioning)
+  - [Testing GitHub Actions](https://github.com/Azure/actions/blob/master/docs/Testing-GitHub-Actions.md)
+  - [Publish to Marketplace](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#publish-the-action-to-marketplace)
+  - [Communication guidelines](https://github.com/Azure/actions/blob/master/communication%20guidelines.md)

```

Comment:
NIT: The naming convention of our files seem a little off to me

Reply:
Yeah, true, and for now not all the files are in the docs folder, will make that consistent